objects/src/accounts/storage/mod.rs

```diff
@@ -11,6 +11,9 @@ pub use slot::{StorageSlot, StorageSlotType};
 mod map;
 pub use map::StorageMap;
 
+mod header;
+pub use header::AccountStorageHeader;
```

Comment:
We also need to re-export it [here](https://github.com/0xPolygonMiden/miden-base/blob/8af403b1d1ae3a9278cc54a1fab34297c8c6905c/objects/src/accounts/mod.rs#L29).